modules/ytp-assets-common/src/less/components/categories.less

```diff
@@ -60,4 +60,13 @@
       }
     }
   }
+
+  a {
+    color: #2a6ebb;
+    font-size: 16px;
```

Comment:
It would be great if font-size and line-height was also defined as a variable somewhere in less-variables